TwitchLib.Api.Helix/Streams.cs

```diff
@@ -97,6 +97,37 @@ public Task<GetStreamsMetadataResponse> GetStreamsMetadataAsync(string after = n
 
             return TwitchGetGenericAsync<GetStreamsMetadataResponse>("/streams/metadata", ApiVersion.Helix, getParams);
         }
+
+        public Task<GetStreamTagsResponse> GetStreamTagsAsync(string broadcasterId, string accessToken = null)
+        {
+            var getParams = new List<KeyValuePair<string, string>>();
+            getParams.Add(new KeyValuePair<string, string>("broadcaster_id", broadcasterId));
+
+            return TwitchGetGenericAsync<GetStreamTagsResponse>("/streams/tags", ApiVersion.Helix, getParams, accessToken);
+        }
+
+        public Task ReplaceStreamTags(string broadcasterId, List<string> tagIds = null, string accessToken = null)
+        {
+            DynamicScopeValidation(AuthScopes.Helix_User_Edit_Broadcast, accessToken);
+
+            var getParams = new List<KeyValuePair<string, string>>();
+            getParams.Add(new KeyValuePair<string, string>("broadcaster_id", broadcasterId));
+
+            string payload = null;
+            if (tagIds != null && tagIds.Count > 0)
+            {
+                payload = "{\"tag_ids\": [";
```

Comment:
Agreed, would be so much clearer.

Reply:
Thought about doing this, but looked at how it's done elsewhere, and it's almost entirely manually.

I guess we can start with this though.

Reply:
Does make it look quite a bit cleaner -->
```
dynamic dynamicPayload = new JObject();
dynamicPayload.tag_ids = new JArray(tagIds);
string payload = dynamicPayload.ToString(Formatting.None);
```

That's all we need!

Reply:
I'm personally not a big fan of delaying it, just because of the fact that it takes way more effort to go through old code just to make those changes, and "it works" so "nobody cares about it". And that pull request will never happen.
Your choice of course.

Reply:
I can do that.